src/plugins/api-proxy/index.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import fs from 'fs';
+import util from 'util';
+import { duotones, getDuotoneUrls } from './util/duotone';
+import getApiProxyRoutes from './routes';
+import proxyApi$ from './proxy';
+
+const API_PROXY_PLUGIN_NAME = 'api-proxy';
+const API_ROUTE_PATH = '/mu_api';
```

Comment:
ah yes, you're right. I think it would make sense in the short term to kill the 'configurable' code, and just `import { API_ROUTE_PATH }` from the api-proxy module into `server-render.jsx` - that's a reasonable dependency requirement.